convex/accessControl.ts

```
@@ -0,0 +1,565 @@
+    };
+  }
+}
+
+/**
+ * Access control middleware wrapper for Convex functions
+ */
+export function withAccessControl(
+  resourceType: ResourceType,
+  requiredPermission: PermissionLevel = "read"
+) {
+  return function<T extends (...args: any[]) => any>(handler: T): T {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)